homeassistant/components/hive/__init__.py

```diff
@@ -52,101 +35,85 @@
     extra=vol.ALLOW_EXTRA,
 )
 
-BOOST_HEATING_SCHEMA = vol.Schema(
-    {
-        vol.Required(ATTR_ENTITY_ID): cv.entity_id,
-        vol.Required(ATTR_TIME_PERIOD): vol.All(
-            cv.time_period, cv.positive_timedelta, lambda td: td.total_seconds() // 60
-        ),
-        vol.Optional(ATTR_TEMPERATURE, default="25.0"): vol.Coerce(float),
-    }
-)
 
-BOOST_HOT_WATER_SCHEMA = vol.Schema(
-    {
-        vol.Required(ATTR_ENTITY_ID): cv.entity_id,
-        vol.Optional(ATTR_TIME_PERIOD, default="00:30:00"): vol.All(
-            cv.time_period, cv.positive_timedelta, lambda td: td.total_seconds() // 60
-        ),
-        vol.Required(ATTR_MODE): cv.string,
-    }
-)
+async def async_setup(hass, config):
+    """Hive configuration setup."""
+    hass.data[DOMAIN] = {"entries": {}}
```

Comment:
It looks like we don't save anything besides config entry specific things in hass.data. Then we don't need the extra layer `entries`.
```suggestion
    hass.data[DOMAIN] = {}
```